src/Core/Localization/Specification/Factory.php

```diff
@@ -103,7 +103,7 @@ public function buildPriceSpecification(
         $currencyDisplayType,
         ?int $maxFractionDigits = null
     ) {
-        $currencyPattern = $cldrLocale->getCurrencyPattern();
+        $currencyPattern = $currency->getPattern($localeCode) ?: $cldrLocale->getCurrencyPattern();
```

Comment:
```suggestion
        $currencyPattern = $currency->getPattern($localeCode) ?? $cldrLocale->getCurrencyPattern();
```

Reply:
Nope `??` only checks "exists an not null" Here we need a value, empty string is not acceptable

Reply:
oh I see :sweat_smile: